pkg/bgp/speaker/speaker.go

```diff
@@ -94,7 +95,7 @@ type MetalLBSpeaker struct {
 }
 
 func (s *MetalLBSpeaker) shutDown() bool {
-	return s.shutdown > 0
+	return atomic.LoadInt32(&s.shutdown) > 0
```

Comment:
Same here

Reply:
technically, these *should* have been part of the PR prior to this one, I caught this and [1] in this PR, I don't hate fixing them too much in these commits, but can re-structure it if you think that's best.

Reply:
Is the idea that you're going to squash some of these in the end into the original PR's commits? If so, then I'm OK with this PR. Otherwise, I'd prefer if the history is cleaned up a bit, such that each commit is reasonably isolated.

Reply:
I dont intend to squash, and typically dont. Ill move the atomic fixes to a new commit which will become the first one in this pull.

Reply:
@christarazi I've pushed a new commit which now is the first commit of this PR which separates out and makes explicit the fixes to the atomic usages.